receiver/sqlserverreceiver/factory_windows.go

```diff
@@ -39,7 +36,17 @@ func createMetricsReceiver(
 		return nil, err
 	}
 
+	var opts []scraperhelper.ScraperControllerOption
+	opts, err = setupScrapers(params, cfg)
+	if err != nil {
+		return nil, err
+	}
+	opts = append(opts, scraperhelper.AddScraper(scraper))
```

Comment:
This is adding the Windows Performance Counter scraper to the list of scrapers in addition to the direct connection scrapers. This allows all logic that is not Windows-specific to be defined once, instead of OS-specific files.